libs/anglify/src/lib/modules/checkbox/interfaces/checkbox.interface.ts

```diff
@@ -1,4 +1,4 @@
-import type { BooleanLike } from '../../utils/interfaces';
+import { BooleanLike } from '../../../utils/interfaces';
 
```

Comment:
Please remove the type `RippleOrigin = 'center' | undefined;`. Use `OverlayRippleOrigin` from `libs/anglify/src/lib/modules/overlay/overlay.interface.ts` instead everywhere.